hydromt/drivers/geodataframe_driver.py

```diff
@@ -18,13 +18,16 @@ class GeoDataFrameDriver(ABC, BaseModel):
     @abstractmethod
     def read(
```

Comment:
No tests were changed, seems that they are missing?

Reply:
Same as above, not looking to test interfaces now.